store/zookeeper/zookeeper.go

```diff
@@ -0,0 +1,357 @@
+package zookeeper
```

Comment:
Not sure what happened here, but fixed it by moving the file first and then modify it (otherwise git would see it as a new file no matter if through `mv` or `git mv` and squash..)